### ISSUE TITLE
Closing Connection Improvement

### ISSUE THREAD

**Issue by spisarau** [mRemoteNG/mRemoteNG#1083]
Sometime it is  happen I open a lot of connections in different folders and subfolders.
## Expected Behavior

When I click right mouse button on the picture of folder I should see item Close all child connections and Close connections in current folder
## Current Behavior
I see only Item Disconnect
## Your Environment

* Version used: ALL
* Operating System and version (e.g. Windows 10 1709 x64):

**Comment by kmscode** [comment]
Could you please clarify? When I use disconnect on a folder, all active connections will be disconnected.

![image](https://user-images.githubusercontent.com/16654335/44746849-8ae9bf80-aad9-11e8-87ea-7ac637d476f1.png)

**Comment by spisarau** [comment]
Yes, in current folder. But if folder has sub-folder, connections in sub-folder will not be closed
